scripts/repo-build-web

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env bash
+
+set -e
+source scripts/package-lib
+
+usage="$0 [OPTION]... RECIPESDIR WORKDIR [REMOTEREPO]
+
+Build new or updated packages defined in RECIPESDIR and create a package index.
+The WORKDIR must be a non-existent directory in which the builds are performed.
+If REMOTEREPO is omitted 'https://toltec.delab.re/testing' is used.
+
+Options:
+
+    -h              Show this help message.
+    -f              Force building all the packages, even those for which
+                    an already built version exists on the remote."
+
+helpflag=
+
+while getopts hf name; do
+    case $name in
+        h) helpflag=1 ;;
+        *) error "Invalid option. Use the -h flag for more information." ;;
+    esac
+done
+
+shift $((OPTIND - 1))
+
+if [[ -n $helpflag ]]; then
+    echo "$usage"
+    exit
+fi
+
+if [[ $# -eq 0 ]]; then
+    error "Missing RECIPESDIR and WORKDIR arguments. Use the -h flag for more information."
+fi
+
+if [[ $# -eq 1 ]]; then
+    error "Missing WORKDIR argument. Use the -h flag for more information."
+fi
+
+if [[ $# -gt 2 ]]; then
+    error "Extraneous arguments. Use the -h flag for more information."
+fi
+
+recipesdir="$1"
+workdir="$2"
```

Comment:
I think this variable should be called `repodir` for consistency with the `repo-build` script and to avoid confusion with the directory that is used for building the individual packages.